web/src/App.js

```diff
@@ -85,7 +85,7 @@ class App extends React.Component {
                     </Menu.Item>
                 }
                 <Menu.Item key='5' style={{float: 'right'}}>
-                  <a target="_blank" without rel="noopener noreferrer" href="https://github.com/casbin/casbin-dashboard">
+                  <a target="_blank" rel="noopener noreferrer" href="https://github.com/casbin/casbin-dashboard">
```

Comment:
Can you please squash your commit, because in this commit you are removing some changes done by you in the previous commit.